CMakeLists.txt

```diff
@@ -199,7 +199,11 @@ if(USE_SYSTEM_ZLIB STREQUAL "OFF")
   message("Using bundled zlib")
 
   if (WIN32)
+  if (CMAKE_BUILD_TYPE MATCHES RelWithDebInfo OR CMAKE_BUILD_TYPE MATCHES Release)
```

Comment:
What's wrong with the previous approach, which used build type == debug? 
With a quick grep I found only these 3 valid values (RelWithDebInfo, Release, Debug)

Reply:
There are more than that. I specifically want to exclude MinSizeRel from these cases.